### DISCUSSION TITLE
No dose in Phantom

### DISCUSSION THREAD

**Discussion by kggalaxy** [nrc-cnrc/EGSnrc#1063] [Questions] [answered]
Why do I do not see any dose inside the phantom? How do I have to change my data?
[Question 2 EGS for github.pdf](https://github.com/nrc-cnrc/EGSnrc/files/12729244/Question.2.EGS.for.github.pdf)

**Comment by rtownson** [comment]
The phantom could be defined incorrectly, for example if the CT ramp used to convert from DICOM to egsphant format was not valid. Or, maybe the ECUT is set high accidentally. If you can switch for a phantom created by dosxyz_gui while holding all the other settings the same, and the simulation looks better, then it is probably an issue with the phantom and the ctcreate process.

**Comment by kggalaxy** [comment]
Dear Dr. Townson,

thans for your answer.
I have fixed the problems.

Best regards
Kirsten Galonske


Am Fr., 29. Sept. 2023 um 18:20 Uhr schrieb Reid Townson <
***@***.***>:

**Reply by rtownson** [reply]
Do you mind sharing the solution, so that others can learn from it? Thanks!

**Comment by kggalaxy** [comment, marked as answer]
Yes, of course. 
I had to uncheck the "Howafarless" -Button.
